tests/wpt/include.ini

```diff
@@ -19,6 +19,12 @@ skip: true
   skip: false
 [fetch]
   skip: false
+  [api]
+    skip: false
+      [cors]
```

Comment:
I think this line (and the subsequent ones) are over-indented, which is throwing off the manifest parser.
```
[api]
  skip: false
  [cors]
    skip: true
  [redirect]
    skip: true
```